charts/sombra/templates/deployment.yaml

```diff
@@ -29,6 +29,7 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      serviceAccountName: {{ include "sombra-chart.serviceAccountName" . }}
```

Comment:
What will this do to existing installations?

Reply:
From my understanding, this should be non-breaking, and should only give new options to specify an existing name

Reply:
Cool yeah, I see your new changes! That makes sense to me